server/fleet/hosts.go

```diff
@@ -1420,7 +1420,7 @@ type HostLite struct {
 	ID                  uint      `db:"id"`
 	TeamID              *uint     `db:"team_id"`
 	Hostname            string    `db:"hostname"`
-	OsqueryHostID       string    `db:"osquery_host_id"`
+	OsqueryHostID       *string   `db:"osquery_host_id"`
```

Comment:
Unrelated, but small fix detected during the POC, `osquery_host_id` can be `NULL` in the DB.